framework/math/quadratures/angular/sldfe_sq_quadrature.cc

```diff
@@ -473,56 +455,55 @@ SimplifiedLDFESQ::Quadrature::IntegrateLDFEShapeFunctions(
   double dx_tilde = (x_tilde_max - x_tilde_min);
   double dy_tilde = (y_tilde_max - y_tilde_min);
 
-  for (int i = 0; i < Nq; ++i)
-  {
-    for (int j = 0; j < Nq; ++j)
-    {
+  for (int i = 0; i < Nq; ++i) {
+    for (int j = 0; j < Nq; ++j) {
       // Determine xy_tilde
-      double x_tilde = x_tilde_min + (1.0 + legendre_qpoints[j][0]) * dx_tilde / 2.0;
-      double y_tilde = y_tilde_min + (1.0 + legendre_qpoints[i][0]) * dy_tilde / 2.0;
+      double x_tilde =
+          x_tilde_min + (1.0 + legendre_qpoints[j][0]) * dx_tilde / 2.0;
+      double y_tilde =
+          y_tilde_min + (1.0 + legendre_qpoints[i][0]) * dy_tilde / 2.0;
       Vector3 xy_tilde(x_tilde, y_tilde, 0.0);
 
       // Map to xyz
-      auto xyz = (sq.rotation_matrix * xy_tilde + sq.translation_vector).Normalized();
+      auto xyz =
+          (sq.rotation_matrix * xy_tilde + sq.translation_vector).Normalized();
 
       // Determine Jacobian
       double r = sqrt(x_tilde * x_tilde + y_tilde * y_tilde + a * a);
       double detJ = (a / (r * r * r)) * dx_tilde * dy_tilde / 4.0;
 
       // Evaluate shape funcs and add to integral
-      for (int k = 0; k < 4; ++k)
-        integral[k] += EvaluateShapeFunction(shape_coeffs[k], xyz) * detJ * legendre_qweights[i] *
-                       legendre_qweights[j];
+      for (int k = 0; k < 4; ++k) {
+        integral[k] += EvaluateShapeFunction(shape_coeffs[k], xyz) * detJ *
+                       legendre_qweights[i] * legendre_qweights[j];
+      }
     } // for j
-  }   // for i
+  } // for i
 
   return integral;
 }
 
-void
-SimplifiedLDFESQ::Quadrature::CopyToAllOctants()
-{
+void SimplifiedLDFESQ::Quadrature::CopyToAllOctants() {
   deployed_SQs.clear(); // just to be sure
   deployed_SQs.reserve(initial_octant_SQs_.size() * 8);
 
   // Define modifying variables
   Vector3 octant_mod(1.0, 1.0, 1.0);
 
   // Top NE octant, no change
-  for (auto& sq : initial_octant_SQs_)
+  for (auto &sq : initial_octant_SQs_)
     deployed_SQs.push_back(sq);
 
   // Top NW octant
   octant_mod = Vector3(-1.0, 1.0, 1.0);
-  for (auto& sq : initial_octant_SQs_)
-  {
+  for (auto &sq : initial_octant_SQs_) {
     SphericalQuadrilateral new_sq = sq;
 
-    for (auto& xyz : new_sq.vertices_xyz)
+    for (auto &xyz : new_sq.vertices_xyz)
       xyz = xyz * octant_mod;
-    auto& vcc = new_sq.centroid_xyz;
+    auto &vcc = new_sq.centroid_xyz;
     vcc = vcc * octant_mod;
-    for (auto& xyz : new_sq.sub_sqr_points)
+    for (auto &xyz : new_sq.sub_sqr_points)
       xyz = xyz * octant_mod;
     new_sq.octant_modifier = octant_mod;
```

Comment:
i think it strange that all of thiese `&` got shifted. likely a choice of formater version issue

it's everywhere in that file, not just here